docker/release/release-ci-aarch64.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Conda does not support CentOS 7 on Linux aarch64, so use CentOS 8
+FROM centos:centos8
```

Comment:
Please do not randomly copy dev image Dockerfile from devtools to other repos.

Side from copying issue, this change *de facto* upgrades release-ci on x64 from CentOS 7 to 8, which as a consequence, will force SecretFlow's minimum `glibc` requirement to `2.28`.

Consider there are still lots of users stuck on distributions with `glibc==2.17`, we cannot approve such change at this time.

Thanks for your contributions.